packages/@okta/vuepress-site/.vuepress/config.js

```diff
@@ -240,7 +240,8 @@ module.exports = {
       exclude: [
         '/test_page/'
       ]
-    }
+    },
+    '@okta/vuepress-plugin-qualtrics-feedback': {}
```

Comment:
This config enables the plugin!
if everything else is good, can we merge it? I don't have the permission to merge into master.

Reply:
Sorry... didn't see that in the initial review